pkgs/applications/version-management/git-and-tools/lab/default.nix

```
@@ -1,27 +1,31 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
-  name = "lab-${version}";
+buildGoModule rec {
+  pname = "lab";
   version = "0.15.2";
 
-  goPackagePath = "github.com/zaquestion/lab";
-
-  subPackages = [ "." ];
-
   src = fetchFromGitHub {
     owner = "zaquestion";
     repo = "lab";
     rev = "v${version}";
     sha256 = "1210cf6ss4ivm2jxq3k3c34vpra02pl91fpmvqbvw5sm53j7xfaf";
   };
 
-  goDeps = ./deps.nix;
+  subPackages = [ "." ];
+
+  modSha256 = "0bw47dd1b46ywsian2b957a4ipm77ncidipzri9ra39paqlv7abb";
```

```suggestion
  modSha256 = "1g72fnkibmdd9i20wgp5fjf556xp2xwan06dmn3f5rkmnicy3ccl";
```

as reported by ofborg.